### PR TITLE
feat: Update instructions in README

### LINKED ISSUE #81: Feature: Steps to edit the `.all-contributorsrc` file
### Type of feature

🍕 Feature

### Current behavior

This repository is using CLI to add contributors to the `README.md` file.

However, there is also a suggestion in the README:

> ... we recommend actually editing the `.all-contributorsrc` file to get more experience with pull requests and reviews.

Does this mean that we are suggesting contributors to add themselves manually in the file? 

### Suggested solution

If we want to encourage contributors to add themselves manually to the file, we also want to provide instructions on how to do so.

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by adiati98**
## Description

This PR adjusts wordings and adds instructions as follow:

- Added a Getting Started section.
- Added a section to manually editing the `.all-contributorsrc` file.
- Added a section to use CLI.
- Added intructions to:
  - Create a new branch.
  - Run `git log` to check commit(s) — because the bot adds and commits the changes automatically, running `git status` won't give us any status information.
  - Add, commit, and push changes.
 -  Added links to GitHub to:
    - Fork, clone, and create pull request.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Fixes #81

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Chandler FRIENDS happy dance GIF

### CODE REVIEW

**Comment by adiati98** [comment]
@BekahHW this PR is ready for review.
Please let me know if I need to fix anything. Thank you 😊

File: README.md [hunk 1]
```diff
@@ -2,26 +2,126 @@
 
 This guestbook is a place for people who have taken [OpenSauced](https://opensauced.pizza)'s [Intro to Open Source course](https://github.com/open-sauced/intro) to take their first steps into contributing to open source.
 
-While there's an automated bot that will help you add yourself to the contributor's list via an issue, we recommend actually editing the `.all-contributorsrc` file to get more experience with pull requests and reviews.
```
Comment (BekahHW) [inline README.md:5]: Instead of using "bot," let's use "CLI tool" throughout the readme.

**Review by BekahHW — CHANGES_REQUESTED** [review]
I think it might be clearer to reorganize this readme a bit.

In the getting started section, give directions from 1-4. After four, write a note.

"There are two ways you can complete this activity to add yourself to the guestbook:

Using the CLI tool
Manually editing the .all-contributorsrc file


Although adding yourself to the list with the CLI tool is simpler, we recommend you to manually editing the file to get more experience with adding and committing your changes, creating pull requests and pull request reviews. Below, you'll find the remaining directions for each method.

### CLI
...start with Run npm run contributors:add 

### Manual Addition
Open the file...

And then instead of saying in each of those sections, see steps 6-9 or 7-9, just copy and past e those steps into the section and remove them from the initial directions.

**Comment by BekahHW** [comment]
@adiati98 have you tested this out? When you use the CLI, it also changes the readme. If they only change the all-contribs file will the automation still happen without the CLI?

**Comment by adiati98** [comment]
@BekahHW I tested everything, but I did miss the generating README step one when we manually edit the file! Thank you for pointing it out! I'll update the docs 😊

**Comment by adiati98** [comment]
> I think it might be clearer to reorganize this readme a bit.
> 
> In the getting started section, give directions from 1-4. After four, write a note.
> 
> "There are two ways you can complete this activity to add yourself to the guestbook:
> 
> Using the CLI tool Manually editing the .all-contributorsrc file
> 
> Although adding yourself to the list with the CLI tool is simpler, we recommend you to manually editing the file to get more experience with adding and committing your changes, creating pull requests and pull request reviews. Below, you'll find the remaining directions for each method.
> 
> ### CLI
> ...start with Run npm run contributors:add
> 
> ### Manual Addition
> Open the file...
> 
> And then instead of saying in each of those sections, see steps 6-9 or 7-9, just copy and past e those steps into the section and remove them from the initial directions.

@BekahHW, I understand that we want to remove the section "How to Add Yourself to the Guestbook" and move it to the "Getting Started" section. But I need some clarifications.

## Q1

Do we want to move the section into a step (step 5) as below:

5. Add yourself to the guestbook.

   There are two ways you can complete this activity:
    
    - Using the CLI tool.
    - Manually editing the `.all-contributorsrc` file
    
    Although adding yourself to the list with the CLI tool is simpler, we recommend you to manually editing the file to get more experience with adding and committing your changes, creating pull requests and pull request reviews. Below, you'll find the remaining directions for each method:
    
    **CLI**
    ...

   **Manual Addition**
    ...

## Q2

OR make a note after step 4 as you suggested:

4. Create a new branch `and add yourself to the guestbook`.

    The best practice for open-source ...

    ```bash
       git checkout -b branch-name
    ```
   
There are two ways you can complete this activity to add yourself to the guestbook:

- Using the CLI tool.
- Manually editing the `.all-contributorsrc` file
    
Although adding yourself to the list with the CLI tool is simpler, we recommend you to manually editing the file to get more experience with adding and committing your changes, creating pull requests and pull request reviews. Below, you'll find the remaining directions for each method:
    
#### CLI
...

#### Manual Addition
...

## Thoughts

I personally lean towards the 1st one as we can make it a step and to avoid confuseness.

**Comment by BekahHW** [comment]
@adiati98 I like the first one as well.

File: README.md (hunk 1 above)
Comment (adiati98) [reply README.md:5]: I've replaced the "bot" with "CLI".

**Comment by adiati98** [comment]
@BekahHW The changes are ready for review.
Please let me know if I need to fix anything. Thank you! 😊

**Review by BekahHW — APPROVED** [review]

**Pull request merged**